commands/functions.go

```diff
@@ -250,34 +251,48 @@ func RunFunctionsList(c *CmdConfig) error {
 	if argCount > 1 {
 		return doctl.NewTooManyArgsErr(c.NS)
 	}
+	var pkg string
+	if argCount == 1 {
+		pkg = c.Args[0]
+	}
 	// Determine if '--count' is requested since we will use simple text output in that case.
 	// Count is mutually exclusive with the global format flag.
 	count, _ := c.Doit.GetBool(c.NS, flagCount)
 	if count && c.Doit.IsSet("format") {
 		return errors.New("the --count and --format flags are mutually exclusive")
 	}
-	// Add JSON flag so we can control output format
-	if !count {
-		c.Doit.Set(c.NS, flagJSON, true)
-	}
-	output, err := RunServerlessExec(actionList, c, []string{flagCount, flagNameSort, flagNameName, flagJSON}, []string{flagLimit, flagSkip})
+	// Retrieve other flags
+	skip, _ := c.Doit.GetInt(c.NS, flagSkip)
+	limit, _ := c.Doit.GetInt(c.NS, flagLimit)
+	nameSort, _ := c.Doit.GetBool(c.NS, flagNameSort)
+	nameName, _ := c.Doit.GetBool(c.NS, flagNameName)
```

Comment:
This is not something new to this PR, but I wonder if we should hide one of these flags as they have the same purpose?

Reply:
The Apache OpenWhisk CLI (`wsk`) just uses `--name-sort`, short form `-n`.   When Adobe issued their CLI (`aio`), it appears that they introduced the "alias" flag `--name` and they moved the `-n` shortcut to it.   By itself, this flag is indeed kind of confusing (seems like it should take a value and allow you to specify a name, somehow).  

When I incorporated parts of `aio` into `nim` I retained the alias behavior.   When I created the `doctl` delegation path I again retained the behavior.   In this PR, in recoding the logic in `golang` I _again_ retained the behavior (copying is easier than thinking).   I think it's appropriate to go back to what `wsk` does, which retains `--name-sort` and `-n` but not `--name`.   This is technically a breaking change but I think it's extremely unlikely to matter to anyone.

@andrewsomething do you agree with that plan?  If so, I'm inclined to just do it (by tacking the change onto this PR).   What do you think?

Reply:
Ach.  Since it is a behavior change, and technically breaking, I decided not to do it as part of this PR so I'm proceeding to merge.   We can think about making the proposed simplification in the future.

Reply:
:+1:  Sounds good.